planning:
  discretization:
    torso_lift_joint      0.10
    shoulder_pan_joint    0.017453292519943295
    shoulder_lift_joint   0.017453292519943295
    upperarm_roll_joint   0.017453292519943295
    elbow_flex_joint      0.017453292519943295
    forearm_roll_joint    0.017453292519943295
    wrist_flex_joint      0.017453292519943295
    wrist_roll_joint      0.017453292519943295
  use_xyz_snap_mprim: false
  use_rpy_snap_mprim: false
  use_xyzrpy_snap_mprim: true
  use_short_dist_mprims: true
  xyz_snap_dist_thresh: 0.0
  rpy_snap_dist_thresh: 0.0
  xyzrpy_snap_dist_thresh: 1.0
  short_dist_mprims_thresh: 0.4
